magefiles/magefile.go

```diff
@@ -178,6 +180,13 @@ func (Local) CleanupGithubOrg() error {
 	return nil
 }
 
+// Deletes Quay repos and robot accounts older than 24 hours with prefixes `has-e2e` and `e2e-demos`, uses env vars DEFAULT_QUAY_ORG and DEFAULT_QUAY_ORG_TOKEN
+func (Local) CleanupQuay() error {
+	quayClient := quay.NewQuayClient(&http.Client{Transport: &http.Transport{}}, utils.GetEnv("DEFAULT_QUAY_ORG_TOKEN", ""), "https://quay.io/api/v1")
+	quayOrg := utils.GetEnv("DEFAULT_QUAY_ORG", "")
```

Comment:
probably better to have pre-run checks for env vars similarly like [here](https://github.com/redhat-appstudio/e2e-tests/pull/421/files#diff-d7a1577b2d546e34aaf816bdd1b83edef28a66a555f049767bafa83268b7efbbR125-R127)

also if the default value for env var is empty, you can use `os.Getenv()`